data/ecosystems/w/whatqiu.toml

```diff
@@ -0,0 +1,25 @@
+# Ecosystem Level Information
+title = "whatqiu"
+
+# Sub Ecosystems
+# These are the titles of other ecosystems in different .toml files in the /data/ecosystems directory
+sub_ecosystems = []
+
+# GitHub Organizations
+# This is a list of links to associated GitHub organizations.
+github_organizations = ["https://github.com/whatqiu"]
```

Comment:
Hi, this is a user and and not an organization.  Can you remove this?

Also, given whatqui is a github user, it's not really an ecosystem.  Can you add the bitcoin oriented repos to the Bitcoin ecosystem instead?

Reply:
Approve of